app/javascript/scavenger_hunt/scenes/Questions/QuestionEleven/QuestionEleven.scss

```diff
@@ -0,0 +1,18 @@
+.letter-box {
```

Comment:
Instead of having letter-box in all the scss file, add it to `styles/scavenger_hunt.scss` and it should be shared by all

Reply:
I am not sure about this design pattern where there is only one .scss for all components. The idea of building costumed component is to have the control over it, however if there is a change in shared scss then it would be difficult to debug. What do you think ?

Reply:
No, that scss is for components sharing common styles. Why repeat same things 11 times?

Reply:
Then we have to keep track of all the components that use that css style. ￼

I think the problem here is that we have similar components that's all. Each component should have their own scss even the style is same as others.